pyreleaseplugin/release.py

```diff
@@ -206,9 +208,10 @@ class ReleaseCommand(Command):
     user_options = [
         ("version=", "v", "new version number"),
         ("version-file=", "f", "a Python file containing the module version number"),
-        ("no-bump-version", "n", "do not bump the version in version-file"),
-        ("description=", "d", "a description of the work done in the release"),        
+        ("no-bump-version", "V", "do not bump the version in version-file"),
```

Comment:
Maybe put the `version` and `no-bump-version` opts next to one another, similar to the `c`/`C` opts?